secret_store/src/key_server_cluster/admin_sessions/key_version_negotiation_session.rs

```diff
@@ -361,8 +391,47 @@ impl<T> SessionImpl<T> where T: SessionTransport {
 		let confirmations = data.confirmations.as_ref().expect(reason);
 		let versions = data.versions.as_ref().expect(reason);
 		if let Some(result) = core.result_computer.compute_result(data.threshold.clone(), confirmations, versions) {
+			// when master node processing decryption service request, it starts with a key versions negotiation session
+			// if negotiation fails, only master node knows about it
+			// => if error is fatal, only master will know about it and report to the contract && request will never be rejected
```

Comment:
`if the error is fatal, only the master ...`

´report it`

`the request will ...`